### PR TITLE
Fix doc string of make_signature() function

### PR DESCRIPTION
**Pull request by srinivasreddy**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/util/_decorators.py [hunk 1]
```diff
@@ -314,14 +314,15 @@ def indent(text, indents=1):
 
 def make_signature(func):
     """
-    Returns a string repr of the arg list of a func call, with any defaults.
+    Returns a string repr of tuple - the arg list of a func call with
```
Comment (WillAyd) [inline pandas/util/_decorators.py]: Short description for docstrings should fit on one line

File: pandas/util/_decorators.py [hunk 2]
```diff
@@ -332,7 +333,7 @@ def make_signature(func):
         n_wo_defaults = len(spec.args) - len(spec.defaults)
         defaults = ('',) * n_wo_defaults + tuple(spec.defaults)
     args = []
-    for i, (var, default) in enumerate(zip(spec.args, defaults)):
+    for var, default in zip(spec.args, defaults):
```
Comment (WillAyd) [inline pandas/util/_decorators.py]: Hmm what are you changing this for?

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/util/_decorators.py (hunk 2 above)
Comment (srinivasreddy) [reply pandas/util/_decorators.py]: There is no need to use enumerate.

Comment (WillAyd) [reply pandas/util/_decorators.py]: Hmm OK thanks. Surprised that isn't picked up by the linter

File: pandas/util/_decorators.py (hunk 1 above)
Comment (srinivasreddy) [reply pandas/util/_decorators.py]: Done

**Review by WillAyd — CHANGES_REQUESTED** [review]
Can you run  `scripts/validate_docstrings.py` against this and make sure no errors?

**Comment by srinivasreddy** [comment]
@WillAyd  Done with the changes. Please take a look again.

**Comment by srinivasreddy** [comment]
I ran `scripts/validate_docstrings.py`,  i didn't introduce any docstring errors.

File: pandas/util/_decorators.py [hunk 3]
```diff
@@ -314,14 +314,14 @@ def indent(text, indents=1):
 
 def make_signature(func):
     """
-    Returns a string repr of the arg list of a func call, with any defaults.
+    Returns a string repr of tuple - the arg list with any defaults and arg list.
```
[the remark this answers is not in the document]
Comment (WillAyd) [reply pandas/util/_decorators.py]: Ah OK. Let's just say `Returns a tuple containing the parameter list with default arguments and the parameter list` or something to the effect. `string repr` is incorrect

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/util/_decorators.py [hunk 4]
```diff
@@ -314,14 +314,14 @@ def indent(text, indents=1):
...
 
     Examples
     --------
-    >>> def f(a,b,c=2) :
+    >>> def f(a,b,c=2):
```
Comment (WillAyd) [inline pandas/util/_decorators.py]: Could still use spacing between parameters here

File: pandas/util/_decorators.py [hunk 5]
```diff
@@ -314,14 +314,14 @@ def indent(text, indents=1):
...
     >>>     return a*b*c
```
Comment (WillAyd) [inline pandas/util/_decorators.py]: And between operators / operands here

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Comment by srinivasreddy** [comment]
@WillAyd  Done

**Comment by WillAyd** [comment]
@srinivasreddy looks like you have a travis failure on the code checks if you can take a look

**Comment by WillAyd** [comment]
You've got a flake8 issue here in `pandas/util/_decorators.py` line 317 for being too long (can also check the flake8 error message above in thread).

**Comment by srinivasreddy** [comment]
@WillAyd  Success. Good to go!!

**Comment by srinivasreddy** [comment]
wait

**Comment by srinivasreddy** [comment]
Done . Merge pls

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @srinivasreddy

**Pull request merged**